### PR TITLE
Add support for tags as labels

### PR DESCRIPTION
**Pull request by jaxxstorm**
Refactored as per @keymone's suggestions.

This will use the arbitrary tags introduced in https://github.com/Yelp/puppet-monitoring_check/commit/f63c963ef4d46d7c6805224d6a75f053dd5cc2bb

We might want to merge this into a another branch and test it before merging this.

### CODE REVIEW

File: files/jira.rb [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 class Jira < BaseHandler
 
+  def build_labels
+    [ "SENSU_#{@event['client']['name']}",
+      "SENSU_#{@event['check']['name']}",
+      "SENSU", *@event['check']['tags'] ].uniq
```
Comment (solarkennedy) [inline files/jira.rb]: What is the significance of the asterisk here?

Hmm. splat? I've never seen this. TIL.

**Comment by solarkennedy** [comment]
How about a quick test case?

**Comment by keymone** [comment]
test case++

if you get any troubles with splat operator - pipe-merging array is also a good option: https://gist.github.com/keymone/5eebf33962282a7b9b0e

**Comment by jaxxstorm** [comment]
Yep I'll try knock some tests together.

**Comment by jaxxstorm** [comment]
rspec added. Let me know what you think @solarkennedy

**Comment by solarkennedy** [comment]
ship

**Pull request merged**